### PR TITLE
Add Notification to block equip in game

### PR DESCRIPTION
**Pull request by tyrosine1153**
### Description

- Add Notification to block equip, unquip or charge AP in game stage
- Extend methods in ItemInformationTooltip, AvatarInfoPopup to add SubmitDisableButton Subject
- Update Localization file(Assets/StreamingAssets/Localization/common.csv) temporarily for Notification Message 

### Related Links
[전투중 장비장착 시도시 장착 불가하다는 한줄팝업 추가하기](https://app.asana.com/0/958521740385861/1201450176916762/f)

### Required Reviewers
@planetarium/9c-dev, @Namyujeong

### Screenshot
image
image
image

Localization file
image

### CODE REVIEW

**Review by sonohoshi — COMMENTED** [review]
Thanks for your contribution!! 
We will check this PR and review or approve it!

File: nekoyume/Assets/_Scripts/UI/Widget/Popup/AvatarInfoPopup.cs [hunk 1]
```diff
@@ -648,6 +654,14 @@ private void ShowTooltip(EquipmentSlot slot)
                         ? L10nManager.Localize("UI_UNEQUIP")
                         : L10nManager.Localize("UI_EQUIP");
                     onSubmit = Equip;
+                    onSubmitDisable = _ =>
+                    {
+                        // ToDo : 상황에 맞게 메세지가 수정되어야 함 -> 일반전투/미미르의 샘/...
```
Comment (sonohoshi) [inline nekoyume/Assets/_Scripts/UI/Widget/Popup/AvatarInfoPopup.cs:659]: Please remove unnecessary TODO comment.

File: nekoyume/Assets/_Scripts/UI/Widget/Popup/AvatarInfoPopup.cs [hunk 2]
```diff
@@ -648,6 +654,14 @@ private void ShowTooltip(EquipmentSlot slot)
...
+                        var msg = inventoryItem.EquippedEnabled.Value
+                            ? L10nManager.Localize("UI_BLOCK_UNEQUIP")
+                            : L10nManager.Localize("UI_BLOCK_EQUIP");;
```
Comment (sonohoshi) [inline nekoyume/Assets/_Scripts/UI/Widget/Popup/AvatarInfoPopup.cs:662]: Please remove unnecessary semicolon.

**Review by sonohoshi — CHANGES_REQUESTED** [review]

File: nekoyume/Assets/StreamingAssets/Localization/common.csv [hunk 3]
```diff
@@ -1618,3 +1618,6 @@ UI_CHARGING_AP,Charging AP...,AP 충전중입니다.,Carregando AP ...,,充電AP
...
 UI_NOT_ENOUGH_HOURGLASS,Insufficient hourglass.,모래시계가 부족합니다.,,,,,,,,,,,,,,,,
 UI_RANKING_CATEGORY_ERROR,Error occured while loading category.,카테고리를 불러오는 도중 오류가 발생했습니다.,Error occured while loading category.,Error occured while loading category.,Error occured while loading category.,Error occured while loading category.,Error occured while loading category.,Error occured while loading category.,Error occured while loading category.,Error occured while loading category.,Error occured while loading category.,Error occured while loading category.,Error occured while loading category.,Error occured while loading category.,Error occured while loading category.,Error occured while loading category.,Error occured while loading category.,
+UI_BLOCK_EQUIP,,장비를 장착할 수 없습니다.,,,,,,,,,,,,,,,,
+UI_BLOCK_UNEQUIP,,장비를 해제할 수 없습니다.,,,,,,,,,,,,,,,,
+UI_BLOCK_CHARGE_AP,,AP를 충전할 수 없습니다.,,,,,,,,,,,,,,,,
```
Comment (Unengine) [inline nekoyume/Assets/StreamingAssets/Localization/common.csv:1623]: I think you should fill english label in order to work fallback localization well.

File: nekoyume/Assets/_Scripts/UI/Widget/Popup/AvatarInfoPopup.cs (hunk 1 above)
Comment (tyrosine1153) [reply nekoyume/Assets/_Scripts/UI/Widget/Popup/AvatarInfoPopup.cs]: I fixed it, thank you!😄

File: nekoyume/Assets/_Scripts/UI/Widget/Popup/AvatarInfoPopup.cs (hunk 2 above)
Comment (tyrosine1153) [reply nekoyume/Assets/_Scripts/UI/Widget/Popup/AvatarInfoPopup.cs]: I fixed it, thank you!😄

File: nekoyume/Assets/_Scripts/UI/Model/ItemInformationTooltip.cs [hunk 4]
```diff
@@ -25,6 +25,7 @@ public class ItemInformationTooltip : Tooltip
 
         public readonly Subject<UI.ItemInformationTooltip> OnSubmitClick = new Subject<UI.ItemInformationTooltip>();
         public readonly Subject<UI.ItemInformationTooltip> OnCloseClick = new Subject<UI.ItemInformationTooltip>();
+        public readonly Subject<UI.ItemInformationTooltip> OnSubmitDisableClick = new Subject<UI.ItemInformationTooltip>();
```
Comment (Unengine) [inline nekoyume/Assets/_Scripts/UI/Model/ItemInformationTooltip.cs:28]: I personally think this variable's naming is not intuitive.
I would recommend `OnClickBlocked`.

Comment (tyrosine1153) [reply nekoyume/Assets/_Scripts/UI/Model/ItemInformationTooltip.cs]: I thought the variable's name should follow `OnClickDisabledSubject` in class `ConditionalButton` .
But, In `ItemInformationTooltip`, Your advice would be better. Thank you!😃I fixed it.

File: nekoyume/Assets/_Scripts/UI/Widget/Popup/AvatarInfoPopup.cs [hunk 5]
```diff
@@ -594,14 +595,16 @@ private void ShowTooltip(InventoryItemView view)
                 return;
             }
 
-            var (submitEnabledFunc, submitText, onSubmit) = GetToolTipParams(view.Model);
+            var (submitEnabledFunc, submitText, onSubmit, onSubmitDisable) = GetToolTipParams(view.Model);
```
Comment (sonohoshi) [inline nekoyume/Assets/_Scripts/UI/Widget/Popup/AvatarInfoPopup.cs:598]:
```suggestion
            var (submitEnabledFunc, submitText, onSubmit, onClickBlocked) = GetToolTipParams(view.Model);
```
In my humble opinion, It's better to unify name with `onClickBlocked` of line 625.

File: nekoyume/Assets/StreamingAssets/Localization/common.csv (hunk 3 above)
Comment (tyrosine1153) [reply nekoyume/Assets/StreamingAssets/Localization/common.csv:1623]: I asked to @Namyujeong for other language. and I'm going to fill it. Can this be okay?

**Review by sonohoshi — APPROVED** [review]
Nice code! If you will add ENG label about `UI_BLOCK_EQUIP`, `UI_BLOCK_UNEQUIP`, `UI_BLOCK_CHARGE_AP`, it will be merge! THX ☺

**Comment by Namyujeong** [comment]
@tyrosine1153 here are the translation text csv. Please unify `UI_BLOCK_EQUIP` and `UI_BLOCK_UNEQUIP` into one.

```
UI_BLOCK_EQUIP,Equipment cannot be changed during battle.,전투 중에는 장비를 변경할 수 없습니다.,O equipamento não pode ser alterado durante a batalha.,,戦闘中に機器を変更することはできません。,Die Ausrüstung kann während der Schlacht nicht geändert werden.,El equipo no se puede cambiar durante la batalla.,อุปกรณ์ไม่สามารถเปลี่ยนแปลงได้ในระหว่างการต่อสู้,Sprzęt nie można zmienić podczas bitwy.,Įranga negali būti pakeista mūšio metu.,Apparatuur kan niet worden gewijzigd tijdens de strijd.,Peralatan tidak dapat diubah selama pertempuran.,L'équipement ne peut pas être changé pendant la bataille.,Оборудование не может быть изменено во время битвы.,គ្រឿងបរិក្ខារមិនអាចត្រូវបានផ្លាស់ប្តូរក្នុងអំឡុងពេលសមរភូមិទេ។,,在战斗中无法改变设备。,A berendezés nem változtatható meg a csatában.
UI_BLOCK_CHARGE_AP,AP Potion cannot be used during battle.,전투 중에는 AP Potion을 사용할 수 없습니다.,AP Poção não pode ser usada durante a batalha.,,APポーションは戦闘中には使用できません。,AP-Trank kann nicht während der Schlacht verwendet werden.,AP Potion no se puede utilizar durante la batalla.,ไม่สามารถใช้ Potion AP ในระหว่างการต่อสู้,AP Mikstura nie może być używana podczas bitwy.,"""Ap Potion"" negalima naudoti mūšio metu.",Ap-drankje kan niet worden gebruikt tijdens de strijd.,Ramuan AP tidak dapat digunakan selama pertempuran.,AP Potion ne peut pas être utilisé pendant la bataille.,А.П. Зелье не может быть использовано во время битвы.,ap potion មិនអាចត្រូវបានប្រើក្នុងអំឡុងពេលសមរភូមិទេ។,,AP药水不能在战斗中使用。,Az AP bájitalt nem használhatják a csatában.
```

**Review by Unengine — APPROVED** [review]

**Review by boscohyun — APPROVED** [review]

**Pull request merged**